your-code/challenge-1.ipynb

```diff
@@ -126,7 +166,59 @@
     "Is also great\n",
     "And would suffice.\"\"\"\n",
     "\n",
-    "# Your code here:\n"
+    "# Your code here:\n",
+    "lst = poem.split()\n",
+    "print(lst)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 56,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'Some': 2, 'say': 3, 'the': 1, 'world': 1, 'will': 1, 'end': 1, 'in': 2, 'fire,': 1, 'ice.': 1, 'From': 1, 'what': 1, 'I’ve': 1, 'tasted': 1, 'of': 2, 'desire': 1, 'I': 3, 'hold': 1, 'with': 1, 'those': 1, 'who': 1, 'favor': 1, 'fire.': 1, 'But': 1, 'if': 1, 'it': 1, 'had': 1, 'to': 1, 'perish': 1, 'twice,': 1, 'think': 1, 'know': 1, 'enough': 1, 'hate': 1, 'To': 1, 'that': 1, 'for': 1, 'destruction': 1, 'ice': 1, 'Is': 1, 'also': 1, 'great': 1, 'And': 1, 'would': 1, 'suffice.': 1}\n"
+     ]
+    }
+   ],
+   "source": [
+    "#here's the dictionary\n",
+    "freq_dict = {i:lst.count(i) for i in lst}\n",
+    "print(freq_dict)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#I could not find the solution to stripping the periods, line breaks and commas just with the strip methods\n",
+    "#I looked for another one, which is in the cell below"
```

Comment:
To solve the problem to remove all punctiation characters I would advice you to use `.replace() ` function
For example
```
words=poem.replace(",", " ").replace(".", " ").replace("\n", " ").split(" ")

poem_dict={x.lower():words.count(x) for x in words if x.isalpha()}
```